views/stylesheet.css

```diff
@@ -0,0 +1,59 @@
+.calculator-container {
+  width: 232px;
+  height: 323px;
+  border-radius: 15px;
+  position: absolute;
+  top:0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  margin: auto;
+}
+
+#display {
```

Comment:
Avoid the use of IDs, unless needed for form labels `<label for="element-id">`.  This should also be namespaced under `calculator`, as defined in the specs: `.calculator-display`